README.md

```diff
@@ -2,3 +2,18 @@
 External trigger functions for the Cylc workflow engine.
 
 <img src="https://travis-ci.org/cylc/cylc-xtriggers.svg?branch=master" alt="build:">
+
+## Copyright and Terms of Use
+
+Copyright (C) 2008-<span actions:bind='current-year'>2020</span> NIWA & British Crown (Met Office) & Contributors.
```

Comment:
@MetRonnie should we use 2021 here? I think otherwise the happy-new-year action would run only 1st january, 2022?

Reply:
(I think I've added a `workflow_dispatch` trigger so you could run it after it gets merged (if it does))